src/liquid/components/ld-icon/readme.md

```diff
@@ -265,6 +265,19 @@ Liquid's icons use the [`currentColor`](https://developer.mozilla.org/en-US/docs
 | `"icon"` | Actual SVG element |
 
 
+## Dependencies
```

Comment:
Have you forgotten to run the build script?

Reply:
Nope, I just added a real non-docs dependency. :)